package.json

```diff
@@ -225,9 +225,9 @@
     "storybook-build": "build-storybook -c .storybook -o docs-ui/.storybook-out --quiet",
     "webpack-profile": "NO_TS_FORK=1 yarn -s webpack --profile --json > stats.json",
     "install-api-docs": "cd api-docs && yarn install",
-    "build-deprecated-docs": "yarn install-api-docs && node api-docs/index.js api-docs/openapi.json tests/apidocs/openapi-deprecated.json",
-    "diff-docs": "yarn install-api-docs && ts-node api-docs/openapi-diff.ts",
-    "deref-api-docs": "node api-docs/index.js tests/apidocs/openapi-spectacular.json tests/apidocs/openapi-derefed.json",
+    "build-deprecated-docs": "yarn install-api-docs && ./node_modules/.bin/ts-node api-docs/index.ts api-docs/openapi.json tests/apidocs/openapi-deprecated.json",
```

Comment:
@nhsiehgit It should be fine to simplify `./node_modules/.bin/ts-node` to `ts-node` here

Reply:
I would have `yarn` resolve the location of `ts-node` by doing `yarn ts-node`